servers/credentialagent/src/entities/users/repos/add-anonymous-user-repo-extension.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2023 Velocity Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const { nanoid } = require('nanoid');
+
+const addAnonymousUserRepoExtension = (parent) => ({
+  addAnonymousUser: () => parent.insert({ _id: `vnf-anonymous-${nanoid()}` }),
```

Comment:
use the same separator as auth0 does for users `|`. I think it should be `anonymous|${nanoid}`

Reply:
done, [10df151](https://github.com/velocitycareerlabs/velocitycore/pull/109/commits/10df15161e201f8492eb20506b607b24578c260e)